CanalTP/navitia-mobility-manager
# Upgrade NMM portal to 1.5

# Description

This PR upgrades NMM Portal to 1.5 release (removing the use of bridges)
Some cleanup is also done in composer dependencies and a helper script is added to ease installation.

## Pull Request type (optional)

This is an upgrade

## How to test

Here are the following steps to test this pull request:

- clone the repository
- adapt parameter.yml and config.nmm_portal.yml
- launch bin/install.sh
- php app/console server:run
- Check NMM is up and running (using admin/admin)
